permits/filters.py

```diff
@@ -61,7 +61,11 @@ class DepartmentPermitRequestFilterSet(BasePermitRequestFilterSet):
         lookup_expr="icontains", label=_("Auteur de la demande"),
     )
     works_object_types__works_object = django_filters.filters.ModelChoiceFilter(
-        queryset=models.WorksObject.objects.order_by("name"),
+        queryset=models.WorksObject.objects.filter(
+            works_object_types__permit_requests__isnull=False
```

Comment:
Is it possible to filer WorksObjet visible for the current user (= displayed in the table) instead of all WorksObject with data application wide?